### ISSUE TITLE
bioSyntax Review Meeting #2

### ISSUE THREAD

**Issue by ababaian** [bioSyntax/bioSyntax#24]
Hey Gang;

Sorry about being AWOL these last couple of weeks. Grant is in and I'm free again!

Let's meet next week and get a definitive plan and timeline in place to do this! I'm stoked to get to do real work again :D. Please add any relevant information to your sections below to update the team

[Meeting Poll](https://dudle.inf.tu-dresden.de/bsmeet2/)

### MozSprint (@alyeffy)
-

### Survey (@Jwong684 @fransilvion) 
-

### Research (@lazypanda10117 @ababaian)
-

### Installers (@alyeffy)
-

### Nano Port (@Ebedthan)
-

### Timeline to completion (draft outline)
- April 10: Release v0.1 release
- April 10: Release Survey
- April 20: Complete written revisions
- April 25: Collect and analyze Survey results
- May 1: Submit Revisions
- May 10-11: Mozilla Sprint

**Comment by ababaian** [comment]
## Clustal Images

For the survey @Jwong684 

![clustal_snp_bw](https://user-images.githubusercontent.com/9220907/38401951-aac32348-390d-11e8-9b65-d2e1ba05e658.jpg)

![clustal_snp_colr](https://user-images.githubusercontent.com/9220907/38401956-af6b76de-390d-11e8-92a4-ecd2328039f3.jpg)

**Comment by Jwong684** [comment]
https://goo.gl/forms/YO89fEPw71JpH3Ac2

I will be sending this survey around my labmates. Let me know if you notice any issues.

**Comment by ababaian** [comment]
Nice =D

My thoughts:

Question 2:
- Instead of `Other:_____________` in Question 2. Can you add an optional Question after asking which other biological file formats do you use, please list them comma separated.
- Alphabetize the options in Question 2

Also both of those images are a little bit difficult to read on my screen as the text is small. Would you be able to make it at least a visible 12pt font on screen?

Question 3:
- Pico/Nano can be one option
- Add `Atom` and `Notepad++` as options

Question 4:
- Also add "Which image do you *prefer*?" as a question.

**Comment by Jwong684** [comment]
In question 2, people have been filling out other biological formats they use already in the Other:______ box, comma separated, so I think it'll be fine. I reordered the options.

I think for the simple purpose of "which image is easier to read?", it gets the point across - and that goes hand-in-hand with "which do you prefer?". There might be the odd user who prefers the image that is less easy to read for whatever reason, but I rather not separate that into two questions. I'll zoom in further with the images.

I've add Pico/nano, atom, and notepad++.

**Comment by ababaian** [comment]
I wouldn't say that the two go hand in hand and they are probably correlated but not dependent on one another. If users find bioSyntax easier to read but still prefer non-bioSyntax then that is exactly the kind of feedback which we need. Forget about the reviewer expectations or trying to show something convincing, what we want is ground truth so that we can improve project development.

There's a fundamental notion in science and that is if your hypothesis cannot be killed then it was never alive to begin with (i.e. a shitty a hypothesis). We need to make sure our hypothesis that there is a user base that prefers bioSyntax and will use it is alive and can be killed. We should find out exactly what kills it.

**Comment by Jwong684** [comment]
Alright, I've included it as an additional question

**Comment by ababaian** [comment]
What do you all think about also asking for emails (optionally at the bottom) so that we could send follow-up questions if they arise? We can promise there will be no promotional or 'newsletter' type emails, just a follow-up question or two if they arise.

**Comment by Jwong684** [comment]
Hey guys,

Try to send out the survey link to any computational biologist/bioinformatician you know. I'm keeping a record of the number of responses in my previous comment. I think a good goal is to get up to 50 people to respond by this week.

**Comment by ababaian** [comment]
So the pilot is good @Jwong684, no-one had any confusion or misunderstanding with any of the questions?

**Comment by Jwong684** [comment]
@ababaian Yes. The questions to test for readability and efficacy were straight-forward to all. There were some questions about what bioSyntax actually was.

**Comment by ababaian** [comment]
That's fair and expected. Sounds good let's do this!

**Comment by ababaian** [comment]
@alyeffy @GuangchuangYu @Ebedthan @fransilvion  if you all can share the link to the survey with people in your circles it would help use hit 50 =D

**Comment by Jwong684** [comment]
Oh wow. We've hit 61 responses so far. I think we could perhaps even get up to 100 then.
